python/tests/compat/test_mantid.py

```diff
@@ -615,5 +615,18 @@ def test_to_workspace_2d_handles_single_x_array():
         assert np.equal(ws.readE(i), np.sqrt(e_vals)).all()
 
 
+@pytest.mark.skipif(not mantid_is_available(),
+                    reason='Mantid framework is unavailable')
+def test_from_mask_workspace():
+    from mantid.simpleapi import LoadMask
+    from os import path
+    dir_path = path.dirname(path.realpath(__file__))
+    mask = LoadMask('HYS', path.join(dir_path, 'HYS_mask.xml'))
+    da = sc.compat.mantid.from_mantid(mask)
+    assert da.data.dtype == sc.dtype.bool
+    assert da.dims == ['spectrum', 'empty']
```

Comment:
Actually I think we should just have `spectrum` here?

Reply:
Yes. SpecialWorkspace types are spectrum-only I believe. Will change.